docs/shp_buildrun_logs.md

```diff
@@ -9,8 +9,7 @@ shp buildrun logs <name> [flags]
 ### Options
 
 ```
-  -F, --follow   Follow the log of a buildrun until it completes or fails.
-  -h, --help     help for logs
+  -h, --help   help for logs
```

Comment:
So addition of log following to `shp buildrun logs` was made as a result of an external user request via https://github.com/shipwright-io/cli/issues/80

Acknowledging that I do see notification of the removal of the feature in the release note, I wonder if that is sufficient notification, and we should cite something in the doc itself.  Admittedly I do not have a confident opinion either way.

I also wonder about how it will look that with `tkn` you can do log following with both Tasks and TaskRuns, but we are removing the analogous flavors of both Build and BuildRuns from `shp`.  And in light of the fact that the early prototypes of log following with `shp` used the `tkn` logic under the covers, and then we moved away from that to facilitate other features, it is unfortunate if we cannot provide both flavors long term.

I'm assuming @otaviof you got to some point in working this issue where you encountered some obstacle you deemed insurmountable.  I know you have been working this for a bit.  But if/when that "ah ha" moment came for you, I missed it previously (distracted with items elsewhere) if you had declared it somehow.

I'll try to see if I construct where the breakdown was when analyzing your changes here, as well as reviewing the error messages in the accompanying issue.

But I would like that we minimally document as concisely as possible where the existing implementation broke down before we pull the trigger on this. i.e. we hit this line of code with one thread and it resulted in this line of code from another thread not doing what we want.

Thanks.

Reply:
@otaviof ^^

Reply:
Another piece of background and context I collected before diving in:  using `tkn logs -F` again for the first time in a while, it looks like with that later versions of `tkn` that provide TaskRun and PIpelineRun log following have changed from my prior recollection, and wait either a) until the job finishes, or b) a non-trivial amount of time (perhaps to avoid any timing windows) before it starts displaying logs.

Maybe such an approach could allow for safe use of -F with buildrun logs.

OK, next time I comment will be after an attempt to understand the changes.

Reply:
I'll take the freedom to resolve this thread, since we will continue on the latest changes.